code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -249,7 +249,8 @@ GLOBAL_LIST_INIT(strippable_monkey_items, create_strippable_list(list(
 /obj/item/organ/brain/tumor/Remove(mob/living/carbon/C, special, no_id_transfer)
 	. = ..()
 	//Removing it deletes it
-	qdel(src)
+	if(!special)
+		qdel(src)
```

Comment:
```
//Special is for instant replacement like autosurgeons
```

What's up with this one?

Reply:
this would cause stuff like double qdel calls

Reply:
cause the remove can get called down the line i don't remember exactly where this happened

Reply:
right i checked Remove can get called during destroy down the line wait wouldnÄ't it suffice to just check QDELETED